src/Resources/views/CRUD/display_time.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{%- spaceless %}
+    {%- if value is empty -%}
+        &nbsp;
+    {%- else -%}
+        {% set options = field_description.options %}
```

Comment:
I don't like this, and here, it's a one-time variable, so it's really unnecessary. Please get rid of that variable.